doc/source/concept_and_workflow/customize_the_boot_process.rst

```diff
@@ -217,6 +217,16 @@ the available kernel boot parameters for these modules:
   with a 480G RAID1 configured for OS deployment. With
   `rd.kiwi.oem.maxdisk=500G`, the deployment is performed on the RAID disk.
 
+``rd.kiwi.oem.skip_resize_check``
+  Specifies to skip the check for unallocated space on a GPT partitioned disk.
+  GPT disks holds a backup table at the end of the disk. Dumping an image to
+  a disk that is larger than the image itself will therefore leave a gab between
+  the backup table and the real end of the disk. {kiwi} uses this situation
+  to detect if there is free space available for a subsequent resize operation.
+  However, smart firmware implementations can also detect this and sometimes
```

Comment:
WOW, I have not yet found myself in this spot. Wondering how on earth a firmware implementation though this could be useful, most likely this is going make the filesystem inconsistent 😞